pep-0594.rst

```diff
@@ -91,67 +93,30 @@ code is not permitted without legal approval.
 * The ``optparse`` and ``getopt`` modules are widely used. They are mature
   modules with very low maintenance overhead.
 * According to David Beazley [5]_ the ``wave`` module is easy to teach to
-  kids and can make crazy sounds. Making a computer generate crazy sounds is
+  kids and can make crazy sounds. Making a computer generate sounds is a
   powerful and highly motivating exercise for a nine-year-old aspiring developer.
   It's a fun battery to keep.
 
 
 Deprecation schedule
 ====================
 
-3.8
----
-
-This PEP targets Python 3.8. Version 3.8.0 final is scheduled to be released
-a few months before Python 2.7 will reach its end of lifetime. We expect that
-Python 3.8 will be targeted by users that migrate to Python 3 in 2019 and
-2020. To reduce churn and to allow a smooth transition from Python 2,
-Python 3.8 will neither raise ``DeprecationWarning`` nor remove any
-modules that have been scheduled for removal. Instead deprecated modules will
-just be *documented* as deprecated. Optionally modules may emit a
-``PendingDeprecationWarning``.
-
-All deprecated modules will also undergo a feature freeze. No additional
-features should be added *unless* python-dev agrees that the deprecation of
-the module is reverted and the code will not be removed. Bug should still be
-fixed.
-
-3.9
----
-
-Starting with Python 3.9, deprecated modules will start issuing
-``DeprecationWarning``. The `parser`_ module is removed and potentially
-replaced with a new module.
-
-All other deprecated modules are fully supported and will receive security
-updates until Python 3.9 reaches its end-of-life. Python 3.9.0 will
-be released about 18 months after 3.8.0 (April 2021?) and most likely
-be supported for five years after the release. The estimated EOL of Python 3.9
-is in 2026.
-
-3.10
+3.11
 ----
 
-In 3.10 all deprecated modules will be removed from the CPython repository
-together with tests, documentation, and autoconf rules.
+Starting with Python 3.11, deprecated modules will start issuing
+``DeprecationWarning``. The estimated EOL of Python 3.11 is October 2027.
```

Comment:
See the earlier, now deleted text, it talked of 3.9 EOL:

> All other deprecated modules are fully supported and will receive security
updates until Python 3.9 reaches its end-of-life. Python 3.9.0 will
be released about 18 months after 3.8.0 (April 2021?) and most likely
be supported for five years after the release. The estimated EOL of Python 3.9
is in 2026. 

Lines 126-130.

Reply:
Right, so the equivalent EOL should be that of Python 3.12, not 3.11, as mentioned above.

Reply:
This is accurate and on purpose. The deprecations would start in Python 3.11 and thus how long people have to notice that the deprecations have occurred. Otherwise it is not an important detail and it can be removed.

Reply:
I'm a little confused, sorry. How does the Python 3.11 _EOL_ affect how long people have to notice that deprecations have occurred? The Python 3.11 EOL doesn't change anything, right, since the situation is exactly the same in Python 3.12 (as you note in the PEP)?

Reply:
I'm with CAM, I feel like the relevant dates are the EOL of 3.10 (last version without warnings) and the EOL of 3.12 (last version where the modules still exist).

Reply:
Since it seems we have consensus here, I've followed this up with PR #2290